fast/stages/1-resman/stage-2-networking.tf

```diff
@@ -60,35 +60,29 @@ module "net-folder" {
     # network security stage 2 service accounts
     var.fast_stage_2.network_security.enabled != true ? {} : {
       "roles/serviceusage.serviceUsageAdmin" = [
-        try(module.nsec-sa-rw[0].iam_email, null)
+        module.nsec-sa-rw[0].iam_email
       ]
       (var.custom_roles["network_firewall_policies_admin"]) = [
-        try(module.nsec-sa-rw[0].iam_email, null)
+        module.nsec-sa-rw[0].iam_email
       ]
       "roles/compute.orgFirewallPolicyUser" = [
-        try(module.nsec-sa-ro[0].iam_email, null)
+        module.nsec-sa-ro[0].iam_email
       ]
       "roles/serviceusage.serviceUsageConsumer" = [
-        try(module.nsec-sa-ro[0].iam_email, null)
+        module.nsec-sa-ro[0].iam_email
       ]
+      # FIXME: this role does not exist. And a try was hiding the error
```

Comment:
I think this is the correct role? the nextsec stage was pretty liberal in IAM handling, I sorted a few things but some might have escaped...

https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/fast-dev/fast/stages/0-bootstrap/data/custom-roles/ngfw_enterprise_viewer.yaml

Reply:
Based on this [PR](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/pull/2484/files#diff-b8235172e37c0123df773c6f33983c692d1f46ecaf9958e510eabdf33ec9ae6aL41-L47) I think `network_firewall_policies_viewer` was actually replaced by `roles/compute.orgFirewallPolicyUser` which is already granted above.